### ISSUE TITLE
template bug with maxconn since PR#564

### ISSUE THREAD

**Issue by a-courteille** [puppetlabs/puppetlabs-haproxy#568]
Hi there, (issue already reported in : https://github.com/puppetlabs/puppetlabs-haproxy/pull/564)
Since the PR 564 is merged, the option template seems to be malfunctioning.
By applying puppet, the following changes are applied on the maxconn option :

```
-  maxconn 150
+  maxconn 0
+  maxconn 1
+  maxconn 2
+  maxconn 3 
+  maxconn 4
+  maxconn 5
+  maxconn 6
...
```
Until maxconn 149
I'm using this call :
```
options => {
  'maxconn'       => 150,
}
```
by trying to set the value to String type
```
options => {
  'maxconn'       => '150',
}
```
The problem seems to be resolved.

I tested the commit before the PR and nothing to report. I think the change of template engine induced this anomaly.
Or maybe it's a normal breaking change ?

Thanks

**Comment by Ramesh7** [comment]
Hi @a-courteille, Thanks for raising issue. The team have worked on this and fix is released in [7.2.0](https://forge.puppet.com/modules/puppetlabs/haproxy).
Looking forward to here on the same. Thanks

**Comment by Ramesh7** [comment]
Closing as haven't heard anything from reporter and also we have pushed the fix and released the same.
